### PR TITLE
'Who Owns and Controls Dogecoin' Article

### PR DESCRIPTION
**Pull request by RichieJohn**
Please check whether links point to valid URL.

### CODE REVIEW

File: content/dogepedia/articles/who-owns-dogecoin.md [hunk 1]
```diff
@@ -5,4 +5,12 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
  
-No owners. Permissionless. Free. For all shibes.
\ No newline at end of file
+One of the wonderful things about Dogecoin is that no-one, or no corporate entity, “owns” the Dogecoin blockchain: it is entirely open-source, permissionless, decentralised, and peer-to-peer.  
+
+The Dogecoin network is “controlled” by the network of nodes through which it operates. Anyone can run a node, and to learn how to do so, [click here](https://dogecoin.com/dogepedia/how-tos/operating-a-node). Some nodes also [mine Dogecoin](https://dogecoin.com/dogepedia/how-tos/mining-dogecoin), meaning they perform computational work as part of the process to both confirm all transactions up to the next block, and the next block. They are rewarded for this work with a block reward.
```
Comment (MishaBoar) [inline content/dogepedia/articles/who-owns-dogecoin.md:10]:
```suggestion
The Dogecoin network is “controlled” by the network of nodes through which it operates. Anyone can run a node, and to learn how to do so, [click here](/dogepedia/how-tos/operating-a-node). Some nodes also [mine Dogecoin](/dogepedia/how-tos/mining-dogecoin), meaning they perform computational work as part of the process to both confirm all transactions up to the next block, and the next block. They are rewarded for this work with a block reward. 
```

Comment (MishaBoar) [reply content/dogepedia/articles/who-owns-dogecoin.md:10]: Relative paths. Links are fine!

**Review by MishaBoar — CHANGES_REQUESTED** [review]

**Review by RichieJohn — COMMENTED** [review]
Great - will use relative paths in future articles. :)

**Comment by RichieJohn** [comment]
This hasn't been merged yet. Owing to my newbie GitHub status, wondering whether I've done something incorrectly? Apologies if so! @tjstebbing @MishaBoar

**Review by MishaBoar — APPROVED** [review]

**Pull request merged**